lib/xiaomi.js

```diff
@@ -518,6 +518,17 @@ const numericAttributes2Payload = (msg, meta, model, options, dataObject) => {
                 payload.battery = value;
             }
             break;
+        case '1057':
+            if (['ZNCLBL01LM'].includes(model.model)) {
+                const lookup = {
+                    0: 'closing',
+                    1: 'opening',
+                    2: 'stop',
```

Comment:
Instead of `stop`, it should be `stopped` in accordance with the exposes:

```javascript
                exposes.enum('motor_state', ea.STATE, ['stopped', 'opening', 'closing'])
                    .withDescription('Motor state'),
```

Reply:
oh yes, fixed. thanks for spotting it